ros2_batch_job/osx_batch/__init__.py

```diff
@@ -38,6 +38,12 @@ def pre(self):
             os.environ['LANG'] = 'en_US.UTF-8'
         if 'ROS_DOMAIN_ID' not in os.environ:
             os.environ['ROS_DOMAIN_ID'] = '111'
+        if 'OPENSSL_ROOT_DIR' not in os.environ:
+            import subprocess
+            brew_openssl_prefix = subprocess.run(
+                ['brew --prefix openssl'], shell=True, stdout=subprocess.PIPE)
+            if not brew_openssl_prefix.stdout is bytes():
+              os.environ['OPENSSL_ROOT_DIR'] = brew_openssl_prefix.stdout.decode().strip('\n')
```

Comment:
Sorry I'm is still experimenting. I think that it depends what we build in the end,  if none of the packages we build use openssl, the build should not fail even with that env variable empty.
So I think that defaulting to not setting it if openssl config is not found is an ok behaviour. I can add a warning or something though to let the dev know that it was not bound.